appcli/__init__.py

```diff
@@ -13,3 +13,4 @@
 from appcli.cli_builder import create_cli
 from appcli.models.cli_context import CliContext
 from appcli.models.configuration import Configuration, Hooks
+from appcli.orchestrators import DockerComposeOrchestrator, DockerSwarmOrchestrator
```

Comment:
These needed to be exposed so that we can initialise our Configuration objects in external projects:
```
configuration = appcli.Configuration(
        app_name='teraflow',
        docker_image='brightsparklabs/teraflow',
        seed_app_configuration_file=Path(BASE_DIR, 'resources/teraflow.yml'),
        seed_templates_dir=Path(BASE_DIR, 'resources/templates'),
        orchestrator=appcli.DockerComposeOrchestrator(None, None)
    )
```